assets/src/scss/blocks/Accordion/AccordionStyle.scss

```diff
@@ -0,0 +1,148 @@
+.accordion-block {
+  _-- {
+    font-family: var(--font-family-primary);
+  }
+
+  &.is-style-light .accordion-content .accordion-headline {
+    --accordion-block-light-style-- {
+      border-color: transparent;
+      background: var(--beige-100);
+      color: var(--color-text-body);
+
+      &:hover {
+        background: var(--beige-200);
+      }
+    }
+  }
+
+  &.is-style-outline .accordion-content .accordion-headline {
+    border-color: var(--grey-500);
+    background: var(--white);
+    color: var(--grey-900);
+
+    &:hover {
+      background: var(--grey-100);
+    }
+  }
+
+  .accordion-content {
+    display: flex;
+    flex-direction: column;
+
+    .accordion-headline {
+      --accordion-block-dark-style-- {
+        border-color: transparent;
+        background: var(--p4-dark-green-800);
+        color: var(--white);
+
+        &:hover {
+          background: var(--p4-dark-green-900);
+        }
+      }
+      cursor: pointer;
+      position: relative;
+      display: block;
+      padding-top: 16px;
+      padding-bottom: 16px;
+      padding-inline-start: 16px;
+      padding-inline-end: 44px;
+      font-size: $font-size-sm;
```

Comment:
We don't have this variable in the tokens file, can we replace it with the appropriate token?